packages/p2p/src/stores/my-ads-store.js

```diff
@@ -2,16 +2,18 @@ import { action, observable } from 'mobx';
 import { getDecimalPlaces } from '@deriv/shared';
 import { localize } from 'Components/i18next';
 import { buy_sell } from 'Constants/buy-sell';
+import { ad_type } from 'Constants/floating-rate';
 import BaseStore from 'Stores/base_store';
 import { countDecimalPlaces } from 'Utils/string';
-import { decimalValidator, lengthValidator, textValidator } from 'Utils/validations';
+import { decimalValidator, lengthValidator, rangeValidator, textValidator } from 'Utils/validations';
 import { requestWS } from 'Utils/websocket';
 
 export default class MyAdsStore extends BaseStore {
     @observable activate_deactivate_error_message = '';
     @observable advert_details = null;
     @observable adverts = [];
     @observable adverts_archive_period = null;
+
```

Comment:
Unnecessary extra line

Reply:
Removed it. Not sure why prettier is not fixing those